packages/govuk-frontend/src/govuk/settings/_colours-applied.scss

```
@@ -29,9 +29,30 @@ $govuk-text-colour: govuk-colour("black") !default;
 ///
 /// @type Colour
 /// @access public
-
+/// @deprecated Use `$govuk-template-background-colour`
+/// if you intend to match or update the background colour of the whole page,
+/// `govuk-colour('light-grey') otherwise.
 $govuk-canvas-background-colour: govuk-colour("light-grey") !default;
 
+// Output a deprecation warning if $govuk-canvas-background-colour is being overridden
+// Remove in next major version.
+@if $govuk-canvas-background-colour != govuk-colour("light-grey") {
+  @include _warning(
+    "$govuk-canvas-background-colour",
+    "$govuk-canvas-background-colour has been deprecated and will be removed in the next major version."
```

**suggestion** I think we should lead people to the actions they can take to handle the deprecation. Even if it's a bit lengthy, I think we should explain what actions users need to take if they were intending to change the colour.

```suggestion
    "$govuk-canvas-background-colour has been deprecated and will be removed in the next major version. Use `$govuk-template-background-colour` if you want to change the background of the `<html>` element and components that need to match it for visual continuity."
```

Maybe with a check from @calvin-lau-sig7 on how this reads.

---

For consistency, I'd suggest following what we've [written in the changelog](https://github.com/alphagov/govuk-frontend/pull/5790/files#r2006000620).

```suggestion
"$govuk-canvas-background-colour has been deprecated and will be removed in the next major version. Use `$govuk-template-background-colour` if you want to change the background colour of the page and background colour of elements that need to match for visual continuity."
```